Mathlib/Probability/ProbabilityMassFunction/Binominal.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2023 Joachim Breitner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joachim Breitner
+-/
+import Mathlib.Probability.ProbabilityMassFunction.Constructions
+import Mathlib.Data.Nat.Choose.Multinomial
+import Mathlib.Tactic.FinCases
+
+/-!
+# The binominal distribution
+
+This file defines the probability mass function of the binominal distribution, and proves
+it to be equal to `Pmf.bernoulli` for `n = 1`.
+-/
+
+noncomputable section
+
+namespace Pmf
+
+/-- The binomial `Pmf`: The probability of that `i` out of `n` coins come up heads if the
+probability of heads is `p`. -/
+def binominal (p : ENNReal) (h : p ≤ 1) (n : ℕ) : Pmf (Fin (n + 1)) :=
```

Comment:
Do you mean `binomial`? You use one spelling in the docstring, but a different one everywhere else.

Reply:
Binomial of course, according to Wikipedia. Silly me. I blame having worked with nominal logic too long. Will fix. Thanks for catching this!

Reply:
```suggestion
/-- The binomial `Pmf`: The probability of that `i` out of `n` coins come up heads if the
probability of heads is `p`. -/
def binomial (p : ENNReal) (h : p ≤ 1) (n : ℕ) : Pmf (Fin (n + 1)) :=
```